tests/did/create.positive.spec.ts

```diff
@@ -0,0 +1,174 @@
+import {
+    ID_TYPE,
+    NETWORK,
+    DEFAULT_CONTEXT,
+    VERIFICATION_METHOD_TYPES
+} from '../constants';
+import { v4 } from 'uuid';
+import { buildSimpleService } from 'helpers';
+import { test, expect } from '@playwright/test';
+import { StatusCodes } from 'http-status-codes';
+import { MethodSpecificIdAlgo, createVerificationKeys } from '@cheqd/sdk';
+
+test.use({ storageState: 'playwright/.auth/user.json' });
+
+test('[Positive] It can create DID with mandatory properties (Form based + Indy style)', async ({ request }) => {
+    // send request to create DID
+    let response = await request.post(`/did/create`, {
+        data: `network=${NETWORK.TESTNET}&identifierFormatType=${ID_TYPE.BASE58BTC}&` +
+            `verificationMethodType=${VERIFICATION_METHOD_TYPES.Ed25519VerificationKey2020}`,
+        headers: { 'Content-Type': 'application/x-www-form-urlencoded' }
+    });
+    expect(response.status()).toBe(StatusCodes.OK);
```

Comment:
Could you move such check into `expect(response).toBeOk()` cause it checks that error code in 2xx family not only 200

Reply:
Done! ✅